code-challenges.js

```diff
@@ -43,9 +91,33 @@ var myMapNumbers2 = [8, -7, 0, 6, 12]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
-
+// Create a function that takes in 1 argument, which is a string
+const noVowels = (str) => {
```

Comment:
for function naming, general convention to describe the action of the function, so starting with a verb is helpful in understanding intention. For example, something like, "removeVowels" sets the expectation for the logic